### PR TITLE
Switch from syscall to the sys/unix package, closes #9

### PR DESCRIPTION
**Pull request by brmzkw**
The [commit](https://github.com/tftp-go-team/hooktftp/commit/5ce7df67eb1c2b5fc811b8804248549e1c71c78c) is originally from @sspans, but we can't merge it since it required to fix conflicts and he deleted his fork.

### CODE REVIEW

File: src/drop_privileges.go [hunk 1]
```diff
@@ -22,22 +23,20 @@ func DropPrivileges(username string) error {
 		return err
 	}
 
-	// TODO: should set secondary groups too
-	err = syscall.Setgroups([]int{gid})
+	err = unix.Setgroups([]int{gid})
```
Comment (epeli) [inline src/drop_privileges.go]: Does this set the secondary groups?

Comment (brmzkw) [inline src/drop_privileges.go]: Woops, I didn't intend to remove the comment.

Comment (brmzkw) [inline src/drop_privileges.go]: The comment has been reintroduced, and I rebased.

**Comment by sspans** [comment]
Yeah, given that we've switched to golang 1.5 this is a no-brainer.

**Pull request merged**